ndb/tests/conftest.py

```diff
@@ -40,3 +43,28 @@ def reset_state():
     yield
     model.Property._FIND_METHODS_CACHE.clear()
     model.Model._kind_map.clear()
+
+
+@pytest.fixture
+def environ():
+    """Copy of ``os.environ``"""
+    original = os.environ
+    environ = original.copy()
```

Comment:
Maybe not...

~~~~
(gcpdev) chris@opportunity:~/proj/google/clouddev/ndb$ pylint google.cloud.ndb tests | wc -l
2732
~~~~

Reply:
The global is the function, not the import of os.environ, FWIW. Would be an actual problem if environ were a recursive function. "Fixed" anyway.

pylint also complains about line 30, because the arg `environ` masks the global `environ`, which is the fixture. Of course, the named argument, `environ`, is what signals to `pytest` to use the fixture named `environ`. 

Maybe we get bogged down in detail sometimes?